### PR TITLE
New Extension: OSS Browser

### PR DESCRIPTION
**Pull request by 404J**
## Description

Add a new extension that can browser and manage Aliyun OSS files.

## Screencast
oss-browser-1
oss-browser-2

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by 404J** [comment]
If reviewer or someone wants to experience this extension, you can send me email, and I will provide configuration data🎉

File: extensions/oss-browser/src/components/folder.tsx [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+      searchBarAccessory={
+        !props.path ? <BucketDropdown buckets={bucketsState} onBucketChange={onBucketChange} /> : null
+      }
+      actions={
+        <ActionPanel>
+          <CommonActions currentFolder={props.path} refresh={refresh} marks={bookmarksState} />
+        </ActionPanel>
+      }
+    >
+      <List.EmptyView title="No data" />
```
Comment (pernielsentikaer) [inline extensions/oss-browser/src/components/folder.tsx:209]:
```suggestion
      <List.EmptyView title="No Data" />
```
You could also include a logo in 64x64 px size 🙂

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪

I tested the extension and have some feedback for you to address:

- Could you make the region preference as a dropdown (if you can find the regions) - This would cause one potential less user error 🙂

CleanShot 2022-08-23 at 08 37 42@2x
 
I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/oss-browser/src/components/folder.tsx (hunk 1 above)
Comment (404J) [reply extensions/oss-browser/src/components/folder.tsx]: A logo? Do you mean the icon when the list is empty? I think the default icon is ok 😅

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**